services/fxart.ts

```diff
@@ -143,15 +121,27 @@ export const allocateClaim = async (body, id) => {
   }
 }
 
-export const updateMetadata = async ({ chain, collection, nft, metadata }) => {
+export const setMetadataUrl = ({ chain, collection, hash }) => {
+  const metadataUrl = new URL(
+    'https://fxart-beta.kodadot.workers.dev/metadata/v2/json',
+  )
+  metadataUrl.searchParams.set('chain', chain)
+  metadataUrl.searchParams.set('collection', collection)
+  metadataUrl.searchParams.set('hash', hash)
+
+  return metadataUrl.toString()
+}
+
+export const updateMetadata = async ({ chain, collection, nft, sn, hash }) => {
```

Comment:
can we please use the `dyndata` format for v2?

EVM uri is resolved by baseURI + sn


https://github.com/OpenZeppelin/openzeppelin-contracts/blob/11dc5e3809ebe07d5405fe524385cbe4f890a08b/contracts/token/ERC721/ERC721.sol#L88-L93

Reply:
Let's use both of them first.
```
evm -> `dyndata`
assethub -> `fxart`
```
Once both of them are stable, let's merge `assethub` into `dyndata`

I still have some work to do on dynamic data on fxart: validation and endpoint migrator (since we moved to find the index to the frontend side, there is a case when the user can accidentally miss the update metadata endpoint). Once the requirements are finalised on `fxart`, let's move that to `dyndata`.

I think even on `dyndata`, it is better to separate them with prefix endpoints to reduce complexity in one endpoint. For example:
```
dyndata.dev/v1/metadata/evm
dyndata.dev/v1/metadata/assethub
```